src/components/workspace/settings/BillingOverview.vue

```diff
@@ -76,10 +76,10 @@
       <div class="billing-card__info-section">
         <div class="billing-card__label">
           {{ $t('billing.volume') }}
-          <PositiveButton
-            v-if="isEventsLimitExceeded"
-            :content="$t('billing.boost') + '!'"
-          />
+<!--          <PositiveButton-->
+<!--            v-if="isEventsLimitExceeded"-->
+<!--            :content="$t('billing.boost') + '!'"-->
+<!--          />-->
```

Comment:
I think this change is not related because you're disabling the BillingOverview page at all, isn't it?